plugins/bulk_upload/xml/batch/BulkUploadEngineXml.php

```diff
@@ -1097,7 +1103,7 @@ protected function sendItemUpdateData($entryId, $entry, array $flavorAssets, arr
 				KBatchBase::$kClient->thumbAsset->setContent($existingthumbAssets[$thumbParamsId], $thumbAssetsResource);
 			}
 			if (strpos($thumbAssetsResource->tags, self::DEFAULT_THUMB_TAG) !== false)
-				KBatchBase::$kClient->thumbAsset->setAsDefault($thumAsset->id);
+				KBatchBase::$kClient->thumbAsset->setAsDefault($thumbsAsset->id);
```

Comment:
$thumbsAsset->id is not a real attribute with value, it's a magic attribute that place a multi-request place holder in the request.